src/transforms/transformers.ts

```diff
@@ -0,0 +1,18 @@
+import { Context } from "../context";
+import { Transformer } from "../types"
+import { Vue2Transformer } from "./vue2";
+import { Vue3Transformer } from "./vue3";
+import { Svelte3Transformer } from "./svelte3";
+
+export function getTranformer(ctx: Context): Transformer {
+  switch (ctx.options.transformer) {
+    case 'svelte3':
+      return Svelte3Transformer(ctx)
+
+    case 'vue2':
+      return Vue2Transformer(ctx)
+
+    default:
+      return Vue3Transformer(ctx)
```

Comment:
```suggestion
    case 'vue3':
      return Vue3Transformer(ctx)
    default:
      throw new Error('Transformer not exits!')
```

Reply:
no the default transformer *is* the vue3 transformer

Reply:
It using resolved options so it will have resolved default value.

Reply:
You are right